src/series10/FindParticularElement.java

```diff
@@ -0,0 +1,41 @@
+package series10;
+
+import java.util.Scanner;
+
+public class FindParticularElement {
+	static String isParticularNumberPresnt(int[] number) 
+	{
+	
+	for(int i = 0; i < number.length; i++)
+	{
```

Comment:
The code inside the function should have proper indentation. The for loop shouldn't be in the same line as the static function name